packages/ui-kit/src/components/Counter/Counter.stories.tsx

```diff
@@ -28,25 +34,27 @@ const connectedParams = {
   }
 }
 
-const ConnectedCounterTemplate = (args: Story['args']) => {
+const Counter = (args: Story['args']) => {
   const { accessToken } = useStorybookAccessToken(
     axiosInstance,
     process.env.STORYBOOK_PROPEL_ACCESS_TOKEN,
     process.env.STORYBOOK_TOKEN_URL
   )
 
-  if (accessToken === '' || accessToken === undefined) {
+  if (accessToken === '' && args?.query) {
```

Comment:
Boolean() control gives better check not only for empty string but also for null and undefined, even it is set on useStorybookAccessToken customHook as default empty string value. Please consider the below part.

```suggestion
  if (Boolean(accessToken) && args?.query) {
```

Inaddition, 'args?.query' can be an empty oject and in this case it will pass. If there are some mendatory properties on the 'query' object it is better checking them, too, such as 'metric'

Reply:
The purpose of the condition is to split `Connected` and `Static` modes. Since the Storybook development is supposed to be performed only by Propel's devs I think we're safe here. If for some reason some dev passes an empty query object, it will fail with an error
<img width="984" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/340f6a85-7730-431e-861c-350f6ed7ce1d">